onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -1721,6 +1720,71 @@ TEST_F(GraphTransformationTests, BiasGeluTest) {
   ASSERT_TRUE(op_to_count["BiasGelu"] == 1);
 }
 
+TEST_F(GraphTransformationTests, BiasGeluSwitchedInputOrder) {
+  auto model_uri = MODEL_FOLDER "fusion/bias_gelu_fusion_format_2.onnx";
+
+  NameMLValMap feeds;
+  RunOptions run_options;
+  run_options.run_tag = "session/tag";
+
+  // create inputs and outputs
+  OrtValue mlvalue_b_i;
+  std::vector<int64_t> dims_b_i = {3072};
+  int64_t num_elements = 1;
+  for (auto& dim : dims_b_i) {
+    num_elements *= dim;
+  }
+
+  int min_fill_value = -10;
+  int max_fill_value = 10;
+  CreateMLValue<float>(TestCPUExecutionProvider()->GetAllocator(0, OrtMemTypeDefault), dims_b_i,
+                       CreateRandomInputData(static_cast<size_t>(num_elements), min_fill_value, max_fill_value), &mlvalue_b_i);
+  feeds.insert(std::make_pair("B_I", mlvalue_b_i));
+
+  OrtValue mlvalue_a_i;
+  std::vector<int64_t> dims_a_i = {3, 512, 3072};
+  num_elements = 1;
+  for (auto& dim : dims_a_i) {
+    num_elements *= dim;
+  }
+
+  CreateMLValue<float>(TestCPUExecutionProvider()->GetAllocator(0, OrtMemTypeDefault), dims_a_i,
+                       CreateRandomInputData(static_cast<size_t>(num_elements), min_fill_value, max_fill_value), &mlvalue_a_i);
+  feeds.insert(std::make_pair("A_I", mlvalue_a_i));
+
+  std::vector<std::string> output_names;
+  output_names.push_back("C");
+
+  auto run_model_test = [&](TransformerLevel level, std::vector<OrtValue>& fetches) {
+    SessionOptions session_options;
+    session_options.graph_optimization_level = level;
+    session_options.session_logid = "OptimizerTests";
+    InferenceSession session{session_options, GetEnvironment()};
+    ASSERT_TRUE(session.Load(model_uri).IsOK());
+    ASSERT_TRUE(session.Initialize().IsOK());
+
+    RunOptions run_options;
+    auto status = session.Run(run_options, feeds, output_names, &fetches);
+    if (!status.IsOK()) {
+      std::cout << "Run failed with status message: " << status.ErrorMessage() << std::endl;
+    }
+    ASSERT_TRUE(status.IsOK());
+  };
+
+  // run model with and w/o optimizations and compare the results
+  std::vector<OrtValue> vanilla_fetches;
```

Comment:
how about chocolate? :)
maybe `unoptimized_fetches`?

Reply:
Haha ... vanilla usually means plain without any modifications (in this case optimizations) that is why I chose it (not because I like a particular dessert ;) ) But if this is adding to confusion then I will rename it :)